common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -596,7 +600,12 @@
                 sequenceLinks.click(this.goto);
 
                 this.sr_container.focus();
+
+                if (window.chromlessView) {
+                    this.postMessageResize(document.body.offsetHeight);
```

Comment:
Let’s include the width as well or at least rename this message as height in case we send width later.

Reply:
ok, I'll send width and height in one message